lib/ecto.ex

```diff
@@ -595,4 +595,67 @@ defmodule Ecto do
       true
     end
   end
+
+  @doc """
+  Loads `data` into a struct or a map.
+
+  The first argument can be a schema, or a map (of types) and determines the return value:
+  a struct or a map, respectively.
+
+  The second argument `data` specifies fields and values that are to be loaded.
+  It can be a map, a keyword list, or a `{fields, values}` tuple.
+  Fields can be atoms or strings.
+
+  Fields that are not present in the schema (or `types` map) are ignored. Values with incorrect
+  types raise an error.
```

Comment:
thanks, I changed the wording to read: `If any of the values has invalid type, an error is raised.`